README.md

```diff
@@ -22,6 +22,11 @@ To get the source code of our SDKs and samples via **git** just type:
 
 Run "C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe .\Microsoft.WindowsAzure.Mobile.Build.msbuild"
 
+NOTE:
+If you are having trouble building with the above line, you can force MsBuild to use a private packages directory by setting NugetPackageSource appropriately: 
+"C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe /p:NugetPackageSource="C:\git\azure-mobile-services-quickstarts\backend\dotnet\private-packages;https://www.nuget.org/api/v2/" Microsoft.WindowsAzure.Mobile.Build.msbuild"
```

Comment:
You can actually add it to the .msbuild file, where you can probably take advantage of MSBuild properties to use the current directory. Then it's built-in automatically and you don't need to specify it.
<DefaultNugetPackageSource>https://www.nuget.org/api/v2/</DefaultNugetPackageSource>